elasticsearch/lib/elasticsearch-prometheus-exporter/README.md

```diff
@@ -1,168 +0,0 @@
-# Prometheus Exporter Plugin for ElasticSearch
```

Comment:
@lukas-vlcek did this never get cherry-picked to es5.x? or do we actually want to remove this here?

Reply:
I think the README is possibly valid but the plugin is available without vendoring

Reply:
added back in